framework/crosscutting/events/storeKafka.ts

```diff
@@ -0,0 +1,112 @@
+import { Kafka } from 'kafkajs';
+
+import Metadata from './integration/metadata';
+import EventAttiv from './integration/eventAttiv';
+import Attivlogger from '../logging/logger';
+import messages from '../messages/message';
+import IStoreBase from './integration/IStoreBase';
+
+export default class StoreSQS implements IStoreBase {
+  private subscribes: Array<EventAttiv> = [];
+
+  private consumers: Object = {};
+
+  private kafka: any;
+
+  constructor(subscribes: Array<EventAttiv>) {
+    this.subscribes = subscribes;
+  }
+
+  init() {
+    const brokers = [...process.env.KAFKA_BROKERS_ENDPOINT.split(',')];
+
+    this.kafka = new Kafka({
+      clientId: process.env.KAFKA_CLIENT_IDENTIFIER,
+      brokers,
+    });
+
+    this.subscribes.forEach(subscribe => {
+      this.addListener(subscribe.listener, subscribe.name);
+    });
+  }
+
+  /**
+   * Funcao responsavel por inserir um novo registro no topico
+   * @param nameHandler Nome do topico
+   * @param metadata Objeto que deve ser inserido na fila
+   */
+  async send(nameHandler: string, metadata: Metadata) {
```

Comment:
is really necessary to connect/disconnect on every message?

Reply:
There is no real need, but it is a recommended best practice to avoid for example in a broker rebalancing your consumer does not fall or synchronize with kafka brokers.

The idea is precisely to each message sent, establish the connection, send the message and close it so that this process ensures that when you establish the connection again, you will be using all available brokers (and consequently the necessary partitions) and closing the connection ensures you have no "stuck" connections to brokers.